lib/api/etas_api.rb

```diff
@@ -0,0 +1,59 @@
+require "cluster"
+require "clusterable/ht_item"
+require "services"
+require "sinatra"
+
+class EtasAPI < Sinatra::Base
+  set :show_exceptions, ENV["show_sinatra_exceptions"] || false
+  no_matching_data_error = {"application_error" => "no matching data"}.to_json
+
+  get "/ping" do
+    "pong"
+  end
+
+  get "/etas/item_access" do
```

Comment:
I wonder if we might be able to have a more generic name than `/etas/item_access` -- ideally this API wouldn't need to know anything about ETAS, and is just returning information about holdings. 

It might be worth thinking about what the other API routes we need are. We know we need similar (but not identical) kinds of information for resource sharing, section 108, umich ssd access -- but I think we could meet those needs from the same endpoint and with different/additional count info.

The other need is to get a list of all the organizations that hold an item, but without the need for copy counts, to support catalog indexing. I don't know if that's the same endpoint but without an organization parameter or a different endpoint.

Maybe worth discussing some.

Reply:
Renamed the API to `HoldingsAPI` and changed this route to `/item_access`.